src/plays/drawing-board/Readme.md

```diff
@@ -0,0 +1,7 @@
+# Drawing Board
+
```

Comment:
This is an advanced play, Please mention all technical details. Think about it like a blog as in someone will read and understand your work. Also, please add a section about all the new libraries included and the purpose of them.